workflows.tf

```diff
@@ -139,7 +139,7 @@ EOF
   labels = {
     goog-partner-solution = "isol_plb32_0014m00001h34hnqai_by7vmugtismizv6y46toim6jigajtrwh"
   }
-  depends_on = [google_project_service.workflows, google_cloudfunctions2_function.cloud_internal_function]
```

Comment:
why removing the dependency ?

Reply:
Its not needed, as the dependency is implicit by using function URI

Reply:
we had bad experience with implicit dependencies, so prefer to keep it there